sentry-spring/src/test/kotlin/io/sentry/spring/EnableSentryTest.kt

```diff
@@ -92,6 +94,16 @@ class EnableSentryTest {
             }
     }
 
+    @Test
+    fun `configures custom TransportFactory`() {
+        ApplicationContextRunner().withConfiguration(UserConfigurations.of(AppConfigWithCustomTransportFactory::class.java))
+            .run {
```

Comment:
This require a note on the docs?

Reply:
It's included in https://github.com/getsentry/sentry-docs/pull/2833